javascript/ql/src/Expressions/ExprHasNoEffect.ql

```diff
@@ -7,10 +7,9 @@
  * @id js/useless-expression
  * @tags quality
  *       maintainability
- *       correctness
+ *       useless-code
```

Comment:
I went for `correctness` for go. I think that is the more interesting case, that the quality result makes the user realise that they meant to write something different. The examples in the javascript qhelp show me that there are situations in javascript where a seemingly useless expression actually has side effects, so perhaps the situation is different for javascript.

Reply:
Sure. Like I said, javascript seems to have more side effects than languages I'm more familiar with, so "useless expressions" have some kind of effect more often.